### PR TITLE
Document supported CSS properties

### PR DESCRIPTION
**Pull request by tjwudi**
Add links to source code that shows supported CSS properties. It will be painful if we don't know what is supported.

Given that CSS properties support may be ever changing, I decided to document links to source code directly.

### CODE REVIEW

**Comment by tjwudi** [comment]
Any follow up for this PR? :)

File: docs/Style.md [hunk 1]
```diff
@@ -91,3 +91,11 @@ var List = React.createClass({
 // ... in another file ...
 <List style={styles.list} elementStyle={styles.listElement} />
 ```
+## Supported Properties
+
+You can checkout latest support of CSS Properties in following Links.
+
+- [View Properties](https://github.com/facebook/react-native/blob/72d3d724a3a0c6bc46981efd0dad8f7f61121a47/Libraries/Components/View/ViewStylePropTypes.js)
+- [Image Properties](https://github.com/facebook/react-native/blob/72d3d724a3a0c6bc46981efd0dad8f7f61121a47/Libraries/Image/ImageStylePropTypes.js)
+- [Text Properties](https://github.com/facebook/react-native/blob/72d3d724a3a0c6bc46981efd0dad8f7f61121a47/Libraries/Text/TextStylePropTypes.js)
```
Comment (vjeux) [inline docs/Style.md]: Can you link to the documentation instead of the js files?

http://facebook.github.io/react-native/docs/text.html#style

**Comment by vjeux** [comment]
Sorry about the delay, it was Sunday at 7pm in California time :x 

Thanks for adding it there, it should make it easier to find out about properties. Can you just link to the documentation website instead of the js files?

**Comment by tjwudi** [comment]
@vjeux Sorry I didn't notice there was doc for that, changed :)

**Comment by vjeux** [comment]
You're not the only one that missed it, so this pull request would really help!

**Pull request merged**